docs/pages/includes/diagnostics/tracing-config.yaml

```diff
@@ -0,0 +1,25 @@
+tracing_service:
+  # Turns tracing on. Default is 'no'
+  enabled: yes
+  # The OLTP exporter to send traces to. Possible values:
```

Comment:
I think we can make this comment neater. If I tile this window to one half of my screen on my 4k monitor, the spacing and line breaks look a bit strange:

<img width="748" alt="image" src="https://user-images.githubusercontent.com/25250668/191854521-70a1483d-ff65-4531-b716-0d554612fadd.png">